website/docs/quick-start-guide.md

```diff
@@ -73,6 +65,13 @@ spark-shell \
   --conf 'spark.sql.extensions=org.apache.spark.sql.hudi.HoodieSparkSessionExtension'
 ```
 ```shell
+# Spark 3.0
```

Comment:
why is this Spark 3.0 , when the default build is Spark 3.3?

Reply:
We are adding back support for spark3.0 and hence showing launch command for spark3.0.